security/advancedtls/crl.go

```diff
@@ -516,3 +523,27 @@ func extractCRLIssuer(crlBytes []byte) ([]byte, error) {
 	}
 	return issuer, nil
 }
+
+func hasExpired(crl *x509.RevocationList, now time.Time) bool {
+	return !now.Before(crl.NextUpdate)
+}
+
+// pemCRLPrefix, pemType, and parseRevocationList  come largely from here
+// x509.go:
+// https://github.com/golang/go/blob/e2f413402527505144beea443078649380e0c545/src/crypto/x509/x509.go#L1669-L1690
+// We must first convert PEM to DER to be able to use the new
+// x509.ParseRevocationList instead of the deprecated x509.ParseCRL
+
+// pemType is the type of a PEM encoded CRL.
+var pemType = "X509 CRL"
```

Comment:
nit: This could be defined as a const. Also it could be used in line 502.

Reply:
Changed